pkg/network/testutil/ping.go

```diff
@@ -14,6 +14,22 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// MustPingTCP is a wrapper around PingTCP that guarantees a non-nil
+// connection is returned
+func MustPingTCP(tb require.TestingT, ip net.IP, port int) net.Conn {
+	c := PingTCP(tb, ip, port)
```

Comment:
Since most of the usages will now be using these `Must*` wrappers, what do you think about returning an error from `Ping*`, so that can be captured here? Otherwise, the error gets swallowed.

Reply:
Error should get reported as part of the `assert.NoError` calls in the `Ping*` functions.

Reply:
ah, right!